src/Windowing/OpenToolkit.Windowing.Common/Enums/OpenGLVersion.cs

```diff
@@ -0,0 +1,27 @@
+//
+// OpenGLVersion.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.Windowing.Common
+{
+    /// <summary>
+    /// Describes the OpenGL version you want the window to use.
+    /// </summary>
+    public enum OpenGLVersion
+    {
+        /// <summary>
+        /// Selects core profile OpenGL 3.3
+        /// </summary>
+        OPENGL_3_3_CORE,
+
+        /// <summary>
+        /// Selects compatability profile OpenGL 3.3
+        /// </summary>
+        OPENGL_3_3_COMPAT,
```

Comment:
These enum names don't follow the style guide and also reminds me too much of Java